src/Components/FadeInBox.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box, BoxProps, useDidMount } from "@artsy/palette"
+import styled from "styled-components"
+
+interface FadeInBoxProps extends BoxProps {
+  opacity?: number
+}
+
+export const FadeInBox: React.FC<FadeInBoxProps> = ({ children }) => {
+  const isMounted = useDidMount()
+
+  return <FadeInWrapper opacity={isMounted ? 1 : 0}>{children}</FadeInWrapper>
+}
+
+const FadeInWrapper = styled(Box)<FadeInBoxProps>`
```

Comment:
Rather than writing wrapper components + regular CSS, we should just use props if we can:

```tsx
<Box
  height="100%"
  opacity: isLoading ? 0 : 1,
  style={{
    transition: "opacity 250ms",
  }}
>
```

A good follow up here is to add the `transition` mixin here:

https://github.com/artsy/palette/blob/main/packages/palette/src/elements/Box/Box.tsx#L50

And then we don't need to jump into the `style` prop at all!